ios/PrivacyInfo.xcprivacy

```diff
@@ -10,6 +10,7 @@
 			<key>NSPrivacyAccessedAPITypeReasons</key>
 			<array>
 				<string>C617.1</string>
+				<string>3B52.1</string>
```

Comment:
:question: whats this change about?

Reply:
It kept being added when I built the app. The description says: "Declare this reason to access the timestamps, size, or other metadata of files or directories that the user specifically granted access to, such as using a document picker view controller." I think it's from the image picker.